packages/plugin/src/theia.d.ts

```diff
@@ -5006,6 +5016,133 @@ declare module '@theia/plugin' {
         constructor(base: WorkspaceFolder | string, pattern: string)
     }
 
+    /**
```

Comment:
I believe these changes should exist in `theia-proposed.d.ts` and not `theia.d.ts` as this API is still proposed in vscode and not finalized.

Reply:
The interfaces used by the search api have been moved to theia-proposed.d.ts